pkg/backend/httpstate/backend.go

```diff
@@ -626,7 +626,15 @@ func (b *cloudBackend) currentUser(ctx context.Context) (string, []string, *work
 		return account.Username, account.Organizations, account.TokenInformation, nil
 	}
 	logging.V(1).Infof("no username for access token")
-	return b.client.GetPulumiAccountDetails(ctx)
+	user, org, token, err := b.client.GetPulumiAccountDetails(ctx)
+	if err != nil {
```

Comment:
Sadness, do we need to remap this in every method?

Reply:
Alternatively we could add a `github.com/pulumi/pulumi/pkg/v3/backend/errors` (or some other place/name) package so that we can use it without creating a cycle.

Reply:
That seems better